modules/wizard/wizard.php

```diff
@@ -3,6 +3,8 @@
  * @package Polylang
  */
 
+defined( 'ABSPATH' ) || exit;
```

Comment:
Not needed if we stick to WordPress recommendations.

Reply:
See https://github.com/polylang/polylang/pull/1443#discussion_r1559022178